### ISSUE TITLE
Add Pluto and other Dwarf planets

### ISSUE THREAD

**Issue by ritwik12** [ritwik12/Celestial-bodies-detection#88]
Currently we only have 8 planets and it would be great to include Pluto and other dwarf planets too.

**Comment by 9gl** [comment]
Hey @ritwik12  , I'm interested in contributing to this issue, so before I start working on it, would you mind sparing your time explaining what the issue is about and pointing me to some resources to get started.

**Comment by ritwik12** [comment]
Yeah. so just like we have 8 planets right now for the classification. Our model is not trained for Pluto and Dwarf planets. If we try to give an image of Pluto to our model. It will classify it as wrong.

**Comment by ritwik12** [comment]
Need to add Training images for Pluto, then add test images. Train the model based on those images.

**Comment by SSahas** [comment]
Hello @ritwik12 , 

I'm interested in solving this issue, and I cloned this repository and I collected some pluto images and added them to both training and test data and added "pluto" to retrained_labels.txt and label_image.py and now how to start training ??is this the correct way ??

**Comment by ritwik12** [comment]
@SSahas Please follow the readme.md. I think you also don't need to update `retrained_labels.txt` as that is something done on it's own with this command below.

To train model you need to run ```python retrain.py --bottleneck_dir=bottlenecks --how_many_training_steps=500 --model_dir=inception --summaries_dir=training_summaries/basic --output_graph=retrained_graph.pb --output_labels=retrained_labels.txt --image_dir=./training_data```

**Comment by SSahas** [comment]
hey @ritwik12 , I trained the model for classifying Pluto planet and got the final test accuracy : 87% , I also run the run.py and the webapp is classifying Pluto images correctly but the retrained_graph.py file is greater than 50 mb so I am not able to push it , can I use git LFS now??

**Comment by ritwik12** [comment]
@SSahas You meant [retrained_graph.pb](https://github.com/ritwik12/Celestial-bodies-detection/blob/master/hub/examples/image_retraining/retrained_graph.pb)? The current one in repo is already 83MB

**Comment by SSahas** [comment]
@ritwik 
Yeah , but when I try to push changes to github repo , it says that retrained_graph.pb is greater than 50 mb and also asked to try Git-LFS and when we use Git-LFS, the main file is sent to the Git-LFS server and a pointer file is created which is used to connect to the actual file in the Git-LFS server, I worked with this in the past.

I think we can access the file normally  but in future if someone wants to retrain the model , I don't  know if the retrained_graph.pb file can be modified normally!!!

should I try it??

**Comment by SSahas** [comment]
hey @ritwik12 ,
I opened a pull request called "Train the model for detecting pluto".
I trained the model and the flask app can also able to detect the images as  pluto .
please look in to it.

please tell if there are any things to be changed.

**Comment by the-silent-geek** [comment]
Hey @ritwik12 , I'm interested in contributing to this issue. This is my first time contributing. I want to add a dwarf planet named 'Makemake' to the classification.

**Comment by yuval728** [comment]
Hey, I'm new to open source and would like to contribute.
